### ISSUE TITLE
installation on opensuse

### ISSUE THREAD

**Issue by mzechmeister** [3fon3fonov/trifon#2]
I have the following problem when using check_your_system_linux_debian.sh
```
For which Python version we want to check/install packages?
1) Python2
2) Python3
#? 1
gfortran - yes!
csh - yes!
setuptools - yes!
pip - yes!
numpy - yes!
scipy - yes!
matplotlib - yes!
PyQt5 - not installed! Do you wish to install PyQt5?
1) Yes
2) No
#? 1
[sudo] Passwort für root: 
^CProblem beim Abrufen der Dateien von 'repo-15.0-local'.
Fehler beim Einhängen von 'fs2:/exports/opt_USW/opensuse/leap/15.0/repos' in '/var/tmp/AP_0xpSwMq8': Mounting media failed                                                             
In der Fehlermeldung oben finden Sie einen entsprechenden Hinweis.
Warnung: Repository 'repo-15.0-local' wird aufgrund des obigen Fehlers übersprungen.
Einige der Repositorys konnten aufgrund eines Fehlers nicht aktualisiert werden.
Repository-Daten werden geladen...
Installierte Pakete werden gelesen...
Paket 'python-pyqt5' nicht gefunden.
Paketabhängigkeiten werden aufgelöst...

Keine auszuführenden Aktionen.
```

**Comment by mzechmeister** [comment]
I tried this
```
pip install PyQt5
Collecting PyQt5
  Could not find a version that satisfies the requirement PyQt5 (from versions: )
No matching distribution found for PyQt5
```

And then
```
pip3 install --user PyQt5
```
This worked!

**Comment by mzechmeister** [comment]
Next problem
```
rxvt - not installed! Do you wish to install rxvt?
1) Yes
2) No
#? 1
[sudo] Passwort für root: 
Problem beim Abrufen der Dateien von 'repo-15.0-local'.
Fehler beim Einhängen von 'fs2:/exports/opt_USW/opensuse/leap/15.0/repos' in '/var/tmp/AP_0x6GzSUB': Mounting media failed (mount.nfs: Connection timed out)                           
In der Fehlermeldung oben finden Sie einen entsprechenden Hinweis.
Warnung: Repository 'repo-15.0-local' wird aufgrund des obigen Fehlers übersprungen.
Einige der Repositorys konnten aufgrund eines Fehlers nicht aktualisiert werden.
Repository-Daten werden geladen...
Installierte Pakete werden gelesen...
'rxvt' wurde in den Paketnamen nicht gefunden. Fähigkeiten werden durchsucht.
Keine Anbieter von 'rxvt' gefunden.
Paketabhängigkeiten werden aufgelöst...

Keine auszuführenden Aktionen.
```

I installed it manually with yast.

**Comment by mzechmeister** [comment]
```
python3 gui.py 
===== 2018.11.18 23:11:14 =====
Traceback (most recent call last):
  File "gui.py", line 27, in <module>
    import batman as batman
  File "./addons/batman/__init__.py", line 6, in <module>
    from .transitmodel import *
  File "./addons/batman/transitmodel.py", line 18, in <module>
    from . import _nonlinear_ld
ImportError: ./addons/batman/_nonlinear_ld.so: undefined symbol: PyCObject_Type
```
Seems to be the case for Yes and Yes-local.

**Comment by 3fon3fonov** [comment]
Oh boy I just commented to your other issue.
Ok lets start one by one: 

1. The bash installer works for debian. Is Open Suse Debian based? At least I am not sure you can use the "sudo apt install", which runs in the script.... I have to make a Suse installation file. I may need 1-2 days fro that.

2. Ok, pip3 install --user PyQt5 worked. Are you on Python3 or Python2? I guess Py3.?

3. try "urxvt", but this program is only optional! if not installed the embeded bash shell in the GUI will be "xterm". Ignore this for now.

4. How did you install batman? Locally (option2) or globally (via pip3, option1). If you have selected option 1 then please remove batman from ./addons (because it first looks there and if not compiled correctly will not work)

**Comment by mzechmeister** [comment]
>     1. The bash installer works for debian. Open Suse is not debian based so you cannot use the "sudo apt install", which runs in the script.... I have to make a Suse installation file using "yast". I may need 1-2 days for that.

So provide a opensuse/kde version. "sudo apt install" seems to work sometimes.
Or make it pure python/pip.
Try to avoid root.
 
>     2. Ok, pip3 install --user PyQt5 worked. Are you on Python3 or Python2? I guess Py3.?
>        Then why did you ran the py2 install in the first line?
 
I had to pip3, i.e. python3, since pip didn't work in this case.

>     1. try "urxvt", but this program is only optional! if not installed the embeded bash shell in the GUI will be "xterm". Ignore this for now.

No idea how and where.
 
>     2. Please remove batman from ./addons (because it first looks there and if not compiled correctly will not work) and do the global install. At the moment no idea why not working Locally, maybe is Suse related (it works with Debian and Mac). Was batman already there when you git clone the tool?

Ok, after removing `addons/batman` this problem is solved.

**Comment by 3fon3fonov** [comment]
> So provide a opensuse/kde version. "sudo apt install" seems to work sometimes.
I am working on it with "zypper", lets see how it will work.

> Or make it pure python/pip.

I think for some packages is still better to make a system install (numpy, scipy, PyQt5), but will test with pip. 

> Try to avoid root.
I am, but is not always possible! For Mac OS it is fully "user" install using "homebrew" and it works excellent! However, one still needs "sudo" to install "homebrew" if not already in the system. For your case, in Suse i still didn't get how to make --user install with zypper. I will keep you posted.

**Comment by 3fon3fonov** [comment]
Ok now I added OpenSuse installer which works perfectly fine!
Tested on a clean OpenSuse leap 15, installs everything one may need on Py2 and Py3.

For Py3, at the time this is written, the "dill" problem (see above) is still present. For Py3 one still needs to make the fix in the _dill.py

I may add all the source codes (dill, batman, emcee, celerite, etc.) in the ./deps
directory and run a Local-install. This however, is with low priority for now!
